external/include/protocols/dhcpc.h

```diff
@@ -107,6 +107,18 @@ int dhcp_client_start(const char *intf);
  */
 void dhcp_client_stop(const char *intf);
 
+/**
+ * @brief Set hostname
+ *
+ * @param[in] intf name of interface to stop dhcpc
+ * @param[in] hostname hostname of interface
```

Comment:
The 2nd param is `hostnameq`, but the comment says `hostname`.
Please check it.

Reply:
Thanks I'll modify it